src/test/java/emissary/test/core/junit5/ExtractionTest.java

```diff
@@ -294,6 +296,12 @@ protected void checkAnswers(Element el, IBaseDataObject payload, List<IBaseDataO
         }
     }
 
+    private void checkNullChildTag(String parentTag, String key) {
```

Comment:
Couple of suggestions here:

1. Consider changing the method name to `checkForMissingNameElement` and changing the failure message to say "element" rather than the ambiguous "tag".  (This is more consistent with the other `checkXXX` methods)
2. Consider adding a `String tname` last parameter so the message to indicate which BDO *(TLD, att-1, etc.)* has the malformed meta/nometa tag. (Again, this mirrors the other `checkXXX` methods.)
```suggestion
    private void checkForMissingNameElement(String parentTag, String key, String tname) {
```

Reply:
Added these changes so the error message will be clearer